microprofile/4.1/TCKResults.adoc

```diff
@@ -0,0 +1,64 @@
+:page-layout: certification
+= MicroProfile 4.1 TCK Results Summary
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the MicroProfile TCK results for releases of MicroProfile 4.1.
+
+== Open Liberty 20.0.0.3 MicroProfile 4.1 Certification Summary
+
+MicroProfile 4.0 consists of 8 MicroProfile Specifications and 4 Jakarta EE 8 specfications.  
+
+* MicroProfile Config 2.0
+
+* MicroProfile Fault Tolerance 3.0
+
+* MicroProfile Health 3.1
+
+* MicroProfile JWT Authentication 1.2
+
+* MicroProfile Metrics 3.0
+
+* MicroProfile OpenAPI 2.0
+
+* MicroProfile OpenTracing 2.0
+
+* MicroProfile Rest Client 2.0
+
+* Jakarta Contexts and Dependency Injection 2.0
+
+* Jakarta RESTful Web Services 2.1
+
+* Jakarta JSON Binding 1.0
+
+* Jakarta JSON Processing 1.1
+
+
+Below are the TCK summaries for each specification.
```

Comment:
According to the requirement of MicroProfile 4.1, in order to certify against MicroProfile 4.1. The included eight MicroProfile component specifications must pass the corresponding TCK tests, while passing the included give Jakarta EE 8 specifications are optional.

Reply:
Done